probe-rs/Cargo.toml

```diff
@@ -93,7 +94,7 @@ pretty_assertions = "1"
 test-case = "3"
 test-log = { version = "0.2.16", features = ["trace"] }
 termtree = "0.5"
-insta = { version = "1.38", default-features = false }
+insta = { version = "1.38", default-features = false, features = ["yaml"] }
```

Comment:
Please remove the `yaml` feature here, it's not needed.

Reply:
Thanks, I'm not sure what I goofed up for it to appear...